source/common/upstream/health_checker/health_checker_base.h

```diff
@@ -0,0 +1,126 @@
+#include "envoy/api/v2/core/health_check.pb.h"
+#include "envoy/event/timer.h"
+#include "envoy/runtime/runtime.h"
+#include "envoy/upstream/health_checker.h"
+
+#include "common/common/logger.h"
+
+namespace Envoy {
+namespace Upstream {
+
```

Comment:
Not sure if we want to introduce new namespace here (probably `HealthCheckers` and the dir name will be changed to `health_checkers`? Please help with some suggestions).